conn.go

```diff
@@ -347,6 +376,97 @@ func (conn *Conn) inWorker() {
 	}
 }
 
+// sigWorker runs in an own goroutine, getting signals and handling
```

Comment:
Being overly explicit here: `sigWorker` does not inherently run in its own goroutine. The only caller of `sigWorker` happens to run it in its own goroutine, but there could be another caller that doesn't do the same. Let's just not document `sigWorker` as running in a goroutine.